DOCSP-35137 Update Example Projects title naming (#3120)
mongodb/docs-realm: source/example-projects.txt

.. _realm-examples:

=================================
Atlas Device SDK Example Projects
=================================

.. facet::
  :name: genre
  :values: reference

> .. meta::
>   :description: Explore engineering and expert-provided example projects to learn best 
>                 practices and common development patterns for the Atlas Device SDKs, Atlas
>                 Device Sync, and Realm database.
> 

Reviewer: We'll need to trim this meta description. Search engines typically truncate this at 120 characters on mobile or 160 characters on the web, so we probably have to lose most of this text.
```suggestion
.. meta::
  :description: Example projects for Atlas Device SDK, Atlas Device Sync, and Realm database.
```
